We have our definition for va_list in a separate header file stdarg.h.

--- pgpool-II-4.3.0/src/include/utils/fe_ports.h.orig	2021-12-07 03:47:33.000000000 +0000
+++ pgpool-II-4.3.0/src/include/utils/fe_ports.h	2022-01-09 11:46:18.840443147 +0000
@@ -29,6 +29,7 @@
 #endif
 
 #include <stdlib.h>
+#include <stdarg.h>
 #ifndef FE_PORTS
 #define FE_PORTS
 #include "parser/pg_config_manual.h"
